types/ramda/test/has-tests.ts

```diff
@@ -1,13 +1,51 @@
 import * as R from 'ramda';
 
 () => {
-  const hasName = R.has('name');
-  const a1: boolean = hasName({ name: 'alice' }); // => true
-  const a2: boolean = hasName({ name: 'bob' }); // => true
-  const a3: boolean = hasName({}); // => false
+    const hasName = R.has('name');
+    const a1: boolean = hasName({ name: 'alice' }); // => true
+    const a2: boolean = hasName({ name: 'bob' }); // => true
+    const a3: boolean = hasName({}); // => false
```

Comment:
I noticed that some of the files in the ramda package aren't formatted with the current version of prettier at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.prettierrc.json

To not having unrelated file changes I haven't used prettier for my contributions at `types/ramda/tools.d.ts` and `types/ramda/index.d.ts`. Would it be okay run prettier there or shall this be in another PR?

Reply:
> I noticed that some of the files in the ramda package aren't formatted with the current version of prettier at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.prettierrc.json
> 
> To not having unrelated file changes I haven't used prettier for my contributions at `types/ramda/tools.d.ts` and `types/ramda/index.d.ts`. Would it be okay run prettier there or shall this be in another PR?

If the changes are unrelated to this issue, you should definitely make a new PR if you can! That would be great too